app/bundles/CoreBundle/Views/Helper/builder.html.php

```diff
@@ -36,6 +38,25 @@
             <i class="text-muted"><?php echo $view['translator']->trans('mautic.core.code.mode.token.dropdown.hint'); ?></i>
         </div>
         <div class="builder-toolbar <?php echo $isCodeMode ? 'hide' : ''; ?>">
+            <div class="panel panel-default" id="preview">
```

Comment:
Can we surround this preview code with 

`<?php if ($preview) : ?>` and `<?php endif; ?>` ?

Because it looks like waste of space when there is empty preview with a button that opens new empty tab. Some users will be confused about it and will take them some time to realise that they have to hit save first.

![screen shot 2018-03-30 at 10 29 31](https://user-images.githubusercontent.com/1235442/38131117-1a694c66-3406-11e8-86a0-95da3b828e48.png)

Another question is if the preview would be also helpful for Code Mode. Right now it's hidden for that mode.

What do you think?